package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "migrate": " npx sequelize-cli db:migrate",
+    "migrate-undo": "npx sequelize db:migrate:undo:all"
```

Comment:
Change 'sequelize' to 'sequelize-cli' for this line. The '-cli' tag is missing. Currently, the 'migrate-undo' script fails but I tested it with the '-cli' tag appended and it worked.

Reply:
oh didn't notice that but I've made the changes now